packages/react-core/src/demos/examples/Tabs/ModalTabs.tsx

```diff
@@ -91,16 +84,23 @@ export const ModalTabs: React.FunctionComponent = () => {
             {products.map((product) => (
               <Card
                 isSelectable
-                isSelectableRaised
-                hasSelectableInput
+                isSelected={selectedProduct?.id === product.id}
                 isCompact
                 key={product.id}
                 id={product.name.replace(/ /g, '-')}
-                onClick={onCardClick(product)}
-                onSelectableInputChange={() => onCardClick(product)()}
-                onKeyPress={onCardKeyPress(product)}
               >
-                <CardTitle>{product.name}</CardTitle>
+                <CardHeader
+                  selectableActions={{
+                    selectableActionId: product.id,
+                    selectableActionAriaLabelledby: 'single-selectable-card-example-1',
+                    name: 'single-selectable-card-example',
+                    variant: 'single',
+                    onChange: onCardClick(product),
+                    hasNoOffset: true
```

Comment:
Not a blocker here, but wondering if this would be a usecase for an "actionable" card instead